Source/WebKit/Configurations/WebKit.xcconfig

```diff
@@ -221,6 +221,8 @@ WK_NO_STATIC_INITIALIZERS_Production_NO_ = $(WK_ERROR_WHEN_LINKING_WITH_STATIC_I
 OTHER_LDFLAGS = $(inherited) -iframework $(SDK_DIR)$(SYSTEM_LIBRARY_DIR)/PrivateFrameworks $(UNEXPORTED_SYMBOL_LDFLAGS) $(FRAMEWORK_AND_LIBRARY_LDFLAGS) $(WK_RELOCATABLE_FRAMEWORK_LDFLAGS) $(SOURCE_VERSION_LDFLAGS) $(PROFILE_GENERATE_OR_USE_LDFLAGS) $(WK_NO_STATIC_INITIALIZERS);
 
 REEXPORTED_FRAMEWORK_NAMES = WebKitLegacy;
+// Needed for bincompat (rdar://117360317)
+REEXPORTED_LIBRARY_NAMES[sdk=macosx*] = objc;
```

Comment:
This won't overwrite the above line, and stop re-exporting WebKitLegacy?

Reply:
It does not, and after installing a root with this change, I see both libraries as re-exports:
```
> dyld_info -dependents /System/Library/Frameworks/WebKit.framework/Versions/A/WebKit
/System/Library/Frameworks/WebKit.framework/Versions/A/WebKit [arm64e]:
    -dependents:
        attributes     load path
        re-export      /usr/lib/libobjc.A.dylib
        re-export      /System/Library/Frameworks/WebKit.framework/Versions/A/Frameworks/WebKitLegacy.framework/Versions/A/WebKitLegacy
...
```

Reply:
My bad, didn't notice that these were distinct settings.